### ISSUE TITLE
Fork Request

### ISSUE THREAD

**Issue by inukaze** [AXeL-dev/distract-me-not#78]
Hi there, i found your repository and reading the https://github.com/AXeL-dev/distract-me-not i want to make a request, i want to know if possible you can make a similar extension for Mozilla Firefox & SeaMonkey (This latest is my favorite Internet suite but the extensions i need does not exist for this) .

The main diferences i want to block specific urls like youtube channels, and/or url youtube videos, and well that is my main intention but not block youtube entire site, because i ever need to see a video from youtube, but youtube does not have option for permanently block a channel and never show videos related to that channel.

Basically i need block custom specific urls not entire sites, extensions like adblock plus or leechblock does not work.

**Comment by AXeL-dev** [comment]
Hello, this extension is already available for Mozilla firefox => https://addons.mozilla.org/fr/firefox/addon/distract-me-not. But it only support recent versions of Firefox so in case you're talking about ESR release of Firefox or an old release, then this might be a little bit complicated since the API on old versions of Firefox isn't the same as the actual one.

Otherwise, i saw that you are planing to use the extension to block youtube channels (which may not fully work, because the extension does only block urls & not the internal content of the pages), so just as a suggestion, i thought that maybe the [youtube-viewer](https://github.com/AXeL-dev/youtube-viewer) extension would help you to achieve something similar, by only showing you the youtube content you wanna see & nothing else. I highly encourage you to give it a try & hope that it will fit your needs.
